modules/API/pages/built-in-endpoints.adoc

```diff
@@ -1867,9 +1870,13 @@ To insert vertices or edges, use the xref:API:built-in-endpoints.adoc#_upsert_da
 
 `+GET /graph/{graph_name}/vertices/{vertex_type}+`
 
-This endpoint returns all vertices having the type _`vertex_type`_ in a graph.
+This endpoint returns all vertices having the type `vertex_type` in a graph.
+
+This endpoint requires the xref:user-access:access-control-model.adoc#_data_crud_privileges[`READ_DATA` privilege] on the vertex type.
```

Comment:
can we add a subsection "Required privilege" for related built-in endpoints, just like gsql command, example: https://docs.tigergraph.com/tigergraph-server/current/user-access/user-management#_required_privilege_2

Reply:
It's great to have the link to the privilege directly.

Reply:
I'll add that separately - we have a lot of endpoints so this could be quite a project